addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -1083,31 +1138,8 @@ ListRenderer.include({
     _renderHeader: function () {
         var $thead = this._super.apply(this, arguments);
 
-        if (!this.columnWidths) {
-            if (!this._hasVisibleRecords(this.state)) {
-                var totalWidth = this._getColumnsTotalWidth($thead);
-                this.columns.forEach(function (column) {
-                    let width;
-                    if (column.attrs.absoluteWidth) {
-                        width = column.attrs.absoluteWidth;
-                    } else if (column.attrs.relativeWidth) {
-                        width = ((column.attrs.relativeWidth / totalWidth * 100) + '%');
-                    }
-                    if (width) {
-                        $thead.find('th[data-name=' + column.attrs.name + ']').css('width', width);
-                    }
-                });
-            } else {
-                // if there are records, we force a min-width for fields with an
-                // absolute width to ensure a correct rendering in edition
-                this.columns.forEach(function (column) {
-                    if (column.attrs.absoluteWidth) {
-                        let width = column.attrs.absoluteWidth;
-                        $thead.find('th[data-name=' + column.attrs.name + ']').css('min-width', width);
-                    }
-                });
-            }
-        }
+        // Must compute invisible columns since none is visible at this moment
+        this._computeDefaultWidths($thead[0], true);
```

Comment:
Not sure to understand why we have to do that here. We should talk about this one anyway